packages/ruby_rubocop.rb

```diff
@@ -0,0 +1,79 @@
+# Adapted from Arch Linux ruby-rubocop PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=ruby-rubocop
+
+require 'package'
+
+class Ruby_rubocop < Package
+  description 'A Ruby static code analyzer and formatter'
+  homepage 'https://rubocop.org'
+  version '1.26.1'
+  compatibility 'all'
+  source_url 'SKIP'
+  # source_url 'https://github.com/rubocop/rubocop.git'
+  # git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_armv7l/ruby_rubocop-1.26.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_armv7l/ruby_rubocop-1.26.1-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_i686/ruby_rubocop-1.26.1-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_x86_64/ruby_rubocop-1.26.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '751d173e446bf5e478b5cf75d0d2bcf3ca67e8a0fbb602667c4e3d224e938c5c',
+     armv7l: '751d173e446bf5e478b5cf75d0d2bcf3ca67e8a0fbb602667c4e3d224e938c5c',
+       i686: '3ba763b890c2bb0a39cd07567bc248b7dc4b12242537e817ef1650b4fb2ce6bc',
+     x86_64: '59942e42753e15dafe07c9e522fa0e01483e5e2ca66b7c54a347bef8e58d4b8c'
+  })
+
+  no_fhs
+
+  depends_on 'libyaml'
+  depends_on 'ruby'
+  depends_on 'xdg_base'
+
+  @xdg_config_home = ENV['XDG_CONFIG_HOME']
+  @xdg_config_home = "#{CREW_PREFIX}/.config" if @xdg_config_home.to_s.empty?
+
+  def self.build
+    # system 'bundle install'
+    # system 'rake build'
+  end
+
+  def self.install
+    # system "gem install --build=#{CREW_DEST_DIR} pkg/rubocop-#{version}.gem"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/.config/rubocop"
+
+    # Remove following line after merge of this PR.
+    system "curl -Lsf https://github.com/satmandu/chromebrew/raw/rubocop_yaml/.rubocop.yml -o #{CREW_DEST_PREFIX}/.config/rubocop/config.yml"
+    # Uncomment after merge
+    # system "curl -Lsf https://github.com/skycocker/chromebrew/raw/master/.rubocop.yml -o #{CREW_DEST_PREFIX}/.config/rubocop/config.yml"
+  end
+
+  def self.postinstall
+    @gem_name = name.sub('ruby_', '')
+    system "gem install -N #{@gem_name} --conservative"
+
+    puts "Installing Chromebrew rubocop config file at #{@xdg_config_home}/rubocop/config.yml".lightblue
+    puts 'This can be overridden by a ~/.rubocop.yml'.lightblue
+    FileUtils.mkdir_p "#{@xdg_config_home}/rubocop"
+    # Remove following line after merge of this PR.
+    system "curl -Lsf https://github.com/satmandu/chromebrew/raw/rubocop_yaml/.rubocop.yml -o #{@xdg_config_home}/rubocop/config.yml"
```

Comment:
`downloader 'https://github.com/satmandu/chromebrew/raw/rubocop_yaml/.rubocop.yml', <hash>, "#{@xdg_config_home}/rubocop/config.yml"`?

Reply:
The point is not to use a hash since we are trusting the Chromebrew repo. If the repo is compromised, then the hash for `rubocop.yml` linter config file in the `ruby_rubofop.rb` can get compromised at the same time as the linter config file. So there really isn't any point to doing a further hash check here.

Reply:
Having said that, @supechicken can downloader handle not having a hash provided to it?

Reply:
I believe you can use the keyword `'SKIP'`.

Reply:
Changes made!